neqo-transport/src/crypto.rs

```diff
@@ -916,11 +921,60 @@ impl CryptoStates {
         }
     }
 
+    /// When decrypting Initial packets, there are potentially multiple active versions.
+    /// The `used_pn` range tracks what has been received on the version that was used.
+    /// But if the version changes, the version we select might have a value of 0,
+    /// rather than the actual value, which can cause packet number recovery to fail.
+    /// To avoid that, have the indicated `version` continue from the previous version.
+    /// This only needs to be run once, so run it when getting header protection.
+    fn maybe_continue_initial_rx(&mut self, version: Version) {
+        // Only do this if this version hasn't been used...
+        // This would be better with `is_none_or`, but that needs rust 1.82
```

Comment:
```suggestion
        // This would be better with `is_none_or`, but that needs MSRV >= 1.82
```
Because I search for "MSRV" in the code when it changes, to find spots to modify.